guide/002-listen-and-respond.md

```diff
@@ -2,66 +2,66 @@
 
 > This article is a work-in-progress.
 
-Once you start your bot, Telegram sends you updates.
+Once you start your bot, Telegram sends you updates based on each request that is being made to the bot.
 
 > See also: [official reference](https://core.telegram.org/bots/api#getting-updates)
 
-Messages are one type of update, but not the only one. When users send messages, your bot receives a `MessageUpdate`. It looks like: `{ update_id, message }`. When users make callback queries, it receives `CallbackQueryUpdate`, which looks like `{ update_id, callback_query }`.
+Messages are one type of `update` that can be sent, but not the only one. When users send messages, your bot receives a `MessageUpdate`. It looks like this: `{ update_id, message }`. When users make callback queries, it receives `CallbackQueryUpdate`, which looks like this: `{ update_id, callback_query }`.
 
 Telegraf allows you to listen to these updates using a simple syntax:
 
 ```TS (Node)
 bot.on("message", ctx => {
-	// Use ctx.message
+	// Use ctx.message to access the "message" object
 });
 
 bot.on("callback_query", ctx => {
-	// Use ctx.callbackQuery
+	// Usage: ctx.callbackQuery
 });
 
 bot.on("chat_join_request", ctx => {
-	// Use ctx.chatJoinRequest
+	// Usage: ctx.chatJoinRequest
 });
 ```
 
-But, remember that all messages are not text messages either! Messages can be `text`, `animation`, `photo`, `video`, or they can be service messages like `video_chat_started`, `successful_payment`. These can also be filtered for with filters!
+But, remember that all messages are not text messages either!. Messages can be `text`, `animation`, `photo`, `video`, or they can be service messages like `video_chat_started`, `successful_payment`. These can also be filtered for with filters!
```

Comment:
No full-stop after exclamation. Exclamation mark already marks end of statement.